test/Interop/Cxx/stdlib/use-std-map.swift

```diff
@@ -7,6 +7,8 @@
 
 import StdlibUnittest
 import StdMap
+import std.map
```

Comment:
@egorzhdan am I supposed to just use `StdMap`. If so, that caused problems locally (looking up operator==, I think).